components/feature/findinpage/src/main/java/mozilla/components/feature/findinpage/view/FindInPageBar.kt

```diff
@@ -92,8 +92,6 @@ class FindInPageBar @JvmOverloads constructor(
     }
 
     override fun clear() {
-        queryEditText.hideKeyboard()
```

Comment:
Is there any strong reason why we shouldn't hide the keyboard here?

Reply:
`queryEditText.clearFocus()` will trigger the hiding of the keyboard anyways, so wouldn't it just be duplicated?

Reply:
Ok, I see it's because we are clearing focus after  `queryEditText.clearFocus()`.
Let's add add some test to verify we are `hideKeyboard` when after `clear` is called and when the view doesn't have the focus 👍

Reply:
I will do that!

Reply:
Sorry if this is a stupid question, but how can I verify `hideKeyboard` since it is an extension method?

Reply:
It's not a problem,  it's a really good question:
As `hideKeyboard` is static is bit tricky to test, but you could have an internal function on `FindInPageBar` and that calls the extension function and in a test you can spy on `FindInPageBar` and verify if it was called.

`FindInPageBar`
 ```kotlin
   @VisibleForTesting
    internal fun hideKeyboard(view: View){
        view.hideKeyboard()
    }
```

In the test:

```kotlin
    fun `test`() {
        val findInPageBar = spy(FindInPageBar(testContext))

        findInPageBar.clear()
        verify(findInPageBar).hideKeyboard(findInPageBar)
    }
```

Reply:
Sure, thank you